modRlp/test/org/aion/rlp/RLPElementTest.java

```diff
@@ -0,0 +1,20 @@
+package org.aion.rlp;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.aion.base.util.ByteUtil;
+import org.junit.Test;
+
+public class RLPElementTest {
+
+    @Test(expected = RuntimeException.class)
+    public void testRLPList_wRuntimeException() {
+        RLPList.recursivePrint(null);
```

Comment:
The class is very well covered because of the `RLP.decode2` calls. Only the null checks were missing, and it felt more elegant to put both tests in one class instead of having a class for each with a single test. If you insist I can split them into individual classes. Do you? :)

Reply:
Oh, I see, makes sense... we can leave it this way for now then.

Philosophically, for a class "X" like having a corresponding "XTest.java," but in this case I think it really doesn't make much difference.